zis-aux/test/utils/build/build.sh

```diff
@@ -20,21 +20,21 @@ xlc "-Wa,goff" \
 -o ../bin/shrmem64-test \
 $ZSSAUX/test/utils/src/shrmem64-test.c \
 $COMMON/c/alloc.c \
+$COMMON/c/shrmem64.c \
 $COMMON/c/utils.c \
 $COMMON/c/timeutls.c \
-$COMMON/c/zos.c \
-$ZSSAUX/src/shrmem64.c ; extattr +a ../bin/shrmem64-test
+$COMMON/c/zos.c ; extattr +a ../bin/shrmem64-test
 
 xlc "-Wa,goff" -DSHRMEM64_TEST_TARGET \
 "-Wc,LANGLVL(EXTC99),FLOAT(HEX),agg,exp,list(),so(),goff,xref,gonum,roconst,gonum,ASM,ASMLIB('SYS1.MACLIB'),LP64,XPLINK" \
 "-Wl,ac=1" -I $COMMON/h -I $ZSSAUX/src -I $ZSSAUX/include \
 -o ../bin/shrmem64-target-test \
 $ZSSAUX/test/utils/src/shrmem64-test.c \
 $COMMON/c/alloc.c \
+$COMMON/c/shrmem64.c \
 $COMMON/c/utils.c \
 $COMMON/c/timeutls.c \
-$COMMON/c/zos.c \
-$ZSSAUX/src/shrmem64.c ; extattr +a ../bin/shrmem64-target-test
+$COMMON/c/zos.c ; extattr +a ../bin/shrmem64-target-test
```

Comment:
Should the ```extattr``` command be on it's own line?

Reply:
There is ```;``` (command separator) between ```xlc``` and  ```extattr```.

Reply:
Yes, but it's off the radar for my tired old eyes